hera_cal/tests/test_utils.py

```diff
@@ -0,0 +1,18 @@
+import nose.tools as nt
+from hera_cal.utils import get_HERA_aa
+import numpy as np,sys
+from hera_cal.calibrations import CAL_PATH
+freqs = np.array([0.15])
+
+class Test_utils(object):
```

Comment:
As you mentioned, a lot of the things in the calfile aren't covered by tests.  I'm OK with that if you are.  Some things are officially covered, but not tested all that gracefully though. The first test here only works by defaulting to the hardcoded date/locations file.  Maybe anything more advanced is essentially testing M&C (i.e. retrieving other array configurations from other dates)?  But you could probably test more things than just the number of antennas being returned correctly, e.g., are you getting the lat/lon right (you're not), or whether reading the default cal file with different dates returns the same number of antennas (it shouldn't).